pkg/core/rest/errors/error_handler.go

```diff
@@ -94,27 +62,27 @@ func HandleError(ctx context.Context, response *restful.Response, err error, tit
 				Reason: violation.Message,
 			})
 		}
-	case api_server_types.IsMaxPageSizeExceeded(err):
+	case err == store.ErrorInvalidOffset:
```

Comment:
I know it's a simple error, but we already have similar ones: https://github.com/kumahq/kuma/blob/51bb6190f28e05c4d826372975f47221f92a62c1/pkg/core/rest/errors/errors.go#L3-L25
I think [it](https://github.com/kumahq/kuma/blob/5811eaa876a692855db5fc13da553a70f8f2203c/pkg/core/resources/store/store.go#L146) also should be renamed (to `InvalidOffsetError`) and be a separate error with `Is` method so this case statement would be similar as in other ones:
```go
case errors.Is(err, &store.InvalidOffsetError{}):
```

Reply:
Agreed it'd be good but that's out of scope of this PR